overlord/snapshotstate/backend/backend_test.go

```diff
@@ -436,6 +436,9 @@ func (s *snapshotSuite) TestAddDirToZip(c *check.C) {
 }
 
 func (s *snapshotSuite) TestHappyRoundtrip(c *check.C) {
+	if os.Geteuid() == 0 {
+		c.Skip("this test cannot run as root")
```

Comment:
Not being familiar with this code it would be more useful to say why the test cannot run as root.